src/display/text_layer.js

```diff
@@ -226,7 +530,61 @@ var renderTextLayer = (function renderTextLayerClosure() {
           self._renderTimer = null;
         }, timeout);
       }
-    }
+    },
+    expandTextDivs: function TextLayer_expandTextDivs(expandDivs) {
+      if (!this._enhanceTextSelection || !this._renderingDone) {
+        return;
+      }
+      if (!this.expanded) {
+        expand(this.bounds, this._viewport);
+        this.expanded = true;
```

Comment:
Question: Provided that it would work correctly, would it make sense from a memory usage perspective to clear out `this._bounds` after `expand(...)` has been called (e.g. by doing `this._bounds.length = 0` here)?